tests/test_path.py

```diff
@@ -185,14 +233,34 @@ def test_inexistant_mechanical(mock_filesystem):
 
 
 def test_get_available_ansys_installation(mock_filesystem, mock_awp_environment_variable):
-    assert get_available_ansys_installations() == dict(
-        zip([202, 211, 231], ANSYS_INSTALLATION_PATHS)
-    )
+    available_ansys_installations = get_available_ansys_installations()
+    if sys.platform == "win32":
+        lowercase_available_ansys_installation = {}
+        for key, value in available_ansys_installations.items():
+            lowercase_available_ansys_installation[key] = value.lower()
+        lowercase_ansys_installation_paths = list(
+            map(str.lower, ANSYS_INSTALLATION_PATHS + ANSYS_STUDENT_INSTALLATION_PATHS)
+        )
+        assert lowercase_available_ansys_installation == dict(
+            zip([202, 211, 231] + [-201, -211], lowercase_ansys_installation_paths)
+        )
+    else:
+        assert get_available_ansys_installations() == dict(
+            zip(
+                [202, 211, 231] + [-201, -211],
+                ANSYS_INSTALLATION_PATHS + ANSYS_STUDENT_INSTALLATION_PATHS,
+            )
+        )
```

Comment:
I would have break this tests in one for linux and another for windows.

Reply:
done